pymc/tests/test_distributions_moments.py

```diff
@@ -341,3 +342,22 @@ def test_weibull_moment(alpha, beta, size, expected):
     with Model() as model:
         Weibull("x", alpha=alpha, beta=beta, size=size)
     assert_moment_is_expected(model, expected)
+
+@pytest.mark.parametrize(
+    "mu, s, size, expected",
+    [
+        (1, 1, None, 1),
+        (1, 1, 5, np.full(5, 1)),
+        (np.arange(1, 6), np.arange(1, 6), None, np.arange(1, 6)),
```

Comment:
One of the test conditions should be changed to have a vector `s` and scalar `mu`, to make sure the `broadcast` is being respected with respect to the parameter that does not matter `s`. For instance:

```suggestion
        (2, np.arange(1, 6), None, np.full(5, 2)),
```